.travis.yml

```diff
@@ -0,0 +1,60 @@
+language: scala
+
+sudo: required
+
+dist: trusty
+
+# Install Docker on Travis.  Merely exposing Docker through the Travis
+# settings is insufficient, because the docker-compose tests we use are
+# unable to see it in that case.
+install:
+  - curl -sSL "https://get.docker.com/gpg" | sudo -E apt-key add -
+  - echo "deb https://apt.dockerproject.org/repo ubuntu-trusty main" | sudo tee -a /etc/apt/sources.list
+  - sudo apt-get update
+  - sudo apt-get -o Dpkg::Options::="--force-confdef" -o Dpkg::Options::="--force-confold" --assume-yes install docker-engine
+  - docker version
+
+cache:
+  directories:
+    - "~/.sbt"
+    - "~/.ivy2"
+    - "project/target/resolution-cache"
+    - "project/target/streams"
+    - "api/target"
+    - "calm_adapter/target"
+    - "common/target"
+    - "id_minter/target"
+    - "ingestor/target"
+    - "transformer/target"
+
+matrix:
+  include:
+    - scala: 2.12.2
+      env: PROJECT=common
+    - scala: 2.12.2
+      env: PROJECT=api
+    - scala: 2.12.2
+      env: PROJECT=transformer
+    - scala: 2.12.2
+      env: PROJECT=calm_adapter
+    - scala: 2.12.2
+      env: PROJECT=ingestor
+    - scala: 2.12.2
+      env: PROJECT=id_minter
```

Comment:
I think the Circle logs is correct. If you see https://github.com/wellcometrust/platform-api/blob/master/project/Common.scala#L8 we explicitly declare 2.11.8 as our scala version. Do we need to specify it in the travis.yaml as well since it's defined in our sbt build? Can we not avoid the repetition?

As for the sbt about output, that seems to be run within platform-infra which is not an sbt project, so it doesn't have any scalaVersion defined. Since it's not defined the default for sbt is 2.10 cause sbt is built with 2.10.